src/main/java/com/o19s/es/ltr/stats/LTRStats.java

```diff
@@ -22,6 +22,7 @@
 /**
  * This class is the main entry-point for access to the stats that the LTR plugin keeps track of.
  */
+@Deprecated
```

Comment:
When a class or method is marked as @Deprecated, it's good practice to provide information about what should be used instead. 
Also add information about when it will be removed.

Also check other places with `@Deprecated`